### DISCUSSION TITLE
Problem with save_graphic

### DISCUSSION THREAD

**Discussion by zmladen** [pyvista/pyvista#4584] [Q&A] [answered]
Hello,

i wanted to try pyvista for some projects. I have the problem of getting started. For the given example the show function is working fine. The problem is that the save_graphic function only shows a background image.

```python
import pyvista as pv
from pyvista import examples

earth_alt = examples.download_topo_global()

pl = pv.Plotter()
actor = pl.add_mesh(examples.load_airplane())
pl.add_background_image(examples.mapfile)

pl.save_graphic("img.svg")  

pl.show()
```

I am using python v3.9.6 on windows 10.

Thanks,
zmladen

**Comment by tkoyama010** [comment, marked as answer]
I guess the [example code](https://docs.pyvista.org/version/stable/api/plotting/_autosummary/pyvista.Plotter.save_graphic.html#pyvista.Plotter.save_graphic) is incorrect. Sorry about it and thanks for your report. This code will work correctly.
```python
import pyvista
from pyvista import examples

pl = pyvista.Plotter(off_screen=True)
_ = pl.add_mesh(examples.load_airplane(), smooth_shading=True)
_ = pl.add_background_image(examples.mapfile)
pl.show(auto_close=False)

pl.save_graphic("img.svg") 
pl.close()
```

**Reply by tkoyama010** [reply]
See https://github.com/pyvista/pyvista/pull/4586.

**Reply by zmladen** [reply]
This works now. Thanks!
